src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -166,3 +166,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
 
     with self.argument_context('ams streaming endpoint list') as c:
         c.argument('account_name', id_part=None)
+
+    with self.argument_context('ams live event') as c:
+        c.argument('account_name', account_name_arg_type)
+        c.argument('name', help='The name of the Live Event.')
+        c.argument('parameters', help='Live Event properties needed for creation.')
```

Comment:
'parameters' sounds generic. It seems you need something like 'streaming_protocol' instead.